lisa/tests/kernel/test_bundle.py

```diff
@@ -141,6 +142,13 @@ def add_metric(self, name, data, units=None):
         """
         self.metrics[name] = TestMetric(data, units)
 
+    def display_and_exit(self) -> type(None):
```

Comment:
So while it's just a few lines I don't see the value of having this in `test_bundle.py`.
If a script wants to do that, it's just 2 lines. If it's about exekall, could you make that the actual default goal? That should be pretty generic enough (print the result and exit).

Reply:
It's very similar to TestEnv.from_cli(), just a helper to write a simple script. It's indeed called by exekall when it is asked for, so even if it is just a few lines it needs to sit somewhere in a function.

Why would you want to make that the default goal ? Having the ResultBundle being the result of expressions makes it easier to post-process the DB, and also allows the glue to print the summary and access the ResultBundle without extra steps. The test expressions are not in charge of managing the flow of execution and decide when it is time to exit, and they are also not in charge of deciding when to display the summary of execution.